vcd/datasource_vcd_resource_list.go

```diff
@@ -590,7 +590,18 @@ func orgNetworkListV2(d *schema.ResourceData, meta interface{}) (list []string,
 func getEdgeGatewayList(d *schema.ResourceData, meta interface{}, resType string) (list []string, err error) {
 	client := meta.(*VCDClient)
 
-	org, vdc, err := client.GetOrgAndVdc(d.Get("org").(string), d.Get("vdc").(string))
+	vdcName := d.Get("parent").(string)
+	if vdcName == "" {
+		vdcName = d.Get("vdc").(string)
+	}
+	if vdcName == "" {
+		vdcName = client.Vdc
+	}
+
```

Comment:
Nit. Extra empty line (which actually makes these ifs and their grouping look stranger than they are).